tests/v1/spec_decode/test_dflash_prefix_cache_masking.py

```
@@ -168,3 +168,35 @@ def test_shift_copy_bounded_by_seq_len():
+    shift_draft_block_tables(
+        tables[1],
+        idx_mapping,
+        num_cached_tokens,
+        seq_lens,
+        BLOCK_SIZE,
+        draft_kv_window,
+        update_seq_lens=True,
+    )
+
+    assert seq_lens.item() == draft_kv_window
+    for table, original in zip(tables, originals):
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)